fhttp/http_client.go

```diff
@@ -732,6 +740,17 @@ func (c *FastClient) connect() net.Conn {
 	c.socketCount++
 	var socket net.Conn
 	var err error
+
+	// Resolve the DNS name when making new connections.
+	if c.socketCount > 1 && !c.noResolveEachConn {
+		c.dest, err = resolve(c.hostname, c.port, c.resolve, c.ipAddrUsage)
+		log.Debugf("Hostname %v resolve to ip %v", c.hostname, c.dest)
```

Comment:
nit: put the [c.id] also in new debug logs

Reply:
added